include/NovelRT/Ecs/SparseSet.h

```diff
@@ -121,9 +121,9 @@ namespace NovelRT::Ecs
 
             ConstIterator operator++(int)
             {
-                Iterator tmp = *this;
+                Iterator tmp = reinterpret_cast<Iterator>(*this);
                 ++(*this);
-                return tmp;
+                return reinterpret_cast<ConstIterator>(tmp);
```

Comment:
What happened here?

Reply:
This was failing due to something I tried and eventually removed. I've just reverted this to the copy in `main` so we should be good here.